stape/mod.ts

```
@@ -0,0 +1,98 @@
+export interface State extends Omit<Props, "apiKey"> {
+  api: ReturnType<typeof createHttpClient<StapeClient>>;
+  apiKey?: string;
+}
+
+/**
+ * @title Stape Server-Side Tagging
+ * @description Server-side tagging solution that bypasses ad blockers and ensures GDPR compliance. Track e-commerce events, page views, and custom events directly from your server to multiple advertising platforms.
+ * @category Analytics
+ * @logo https://raw.githubusercontent.com/deco-cx/apps/main/stape/logo.png
+ */
+export default function App(props: Props): App<Manifest, State> {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'App'. Consider to delete it or rename it.

'App' is defined here:


(lint/suspicious/noRedeclare)